drivers/cas_access/app/models/cas_access/filters/filter_base.rb

```diff
@@ -281,33 +283,36 @@ def ordered_steps
         scope = CasAccess::Reporting::Decisions.on_route(match_route)
         step_order = scope.distinct.
           pluck(:match_step, :decision_order).to_h
-        steps = step_order.keys
-        at = scope.arel_table
-        at2 = at.dup
-        at2.table_alias = 'at2'
-        followups = steps.map do |step|
-          followups = scope.where(
-            at2.project(Arel.star).
-              where(at2[:client_id].eq(at[:client_id])).
-              where(at2[:match_id].eq(at[:match_id])).
-              where(at2[:decision_order].lt(at[:decision_order])).
-              where(at2[:match_step].eq(step)).
-              exists,
-          ).distinct.pluck(:match_step, :decision_order).
-            map do |match_step, decision_order|
-              "(#{decision_order}) #{match_step}"
-            end
-          [step, followups]
-        end.to_h
+        step_order.keys
```

Comment:
This is missing some assignment here I think.

Reply:
ah oops, I thought I rolled this change back. Thanks for catching!